Realm/RLMArray.h

```diff
@@ -492,6 +492,18 @@ __attribute__((warn_unused_result));
  */
 - (instancetype)freeze;
 
+/**
+ Returns a live, mutable version of this frozen collection.
+
+ This method resolves a reference to a live copy of the same frozen collection.
+ If a called on a non-frozen collection, will return itself.
```

Comment:
Weird grammar

Reply:
I think we probably just want to call non-frozen collections "live collections"?

The comma is weird too.